lib/common.sh

```diff
@@ -946,8 +946,9 @@ common::set_cloud_binaries () {
     [[ -x $GSUTIL ]] && break
   done
 
-  # gcloud should be in the same place
```

Comment:
Looking back at the original [PR #695](https://github.com/kubernetes/release/pull/695), Michael wanted this to be an [if statement instead of the && shorthand](https://github.com/kubernetes/release/pull/695).

So basically have the same thing as Peter, with the curly brace correction:

```bash
common::set_cloud_binaries () {

  logecho -n "Checking/setting cloud tools: "

  for GSUTIL in "$(which gsutil)" /opt/google/google-cloud-sdk/bin/gsutil; do
    if [[ -x "$GSUTIL" ]]; then
      break
    fi
  done

  for GCLOUD in "${GSUTIL/gsutil/gcloud}" "$(which gcloud)"; do
    if [[ -x "$GCLOUD" ]]; then
      break
    fi
  done
.
.
.
```

Reply:
updated by force push to squash commits